grass7/raster/r.centroids/r.centroids.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
```suggestion
#!/usr/bin/env python
```

The change to `python3` is taken case of by `g.extension`.